zwave_me_server/run.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# If you are running Z-Way in docker without home assistant, specify the path to your interface here
+device=/dev/ttyACM0
+
+# Forcibly update the interface path in the user config configjson-XXXXXXXXXXXXXXXXXX at startup.
+# Do not use this function if more than Z-Wave 1 interface is used
+forceDevice=false
+
+# Default config path
+defCJ=/opt/z-way-server/automation/defaultConfigs/config.json
+
+# If we find file options.json, we use the interface and options from this file
+if [ -f "/data/options.json" ]; then
+  # Get device path from Configuration tab of addon
+  device=$(grep -Eo '/dev/tty[A-Z]*[0-9]' /data/options.json)
+  forceDevice=$(grep "configjson_device_replace" /data/options.json | cut -d ":" -f 2)
+fi
+
+# Change device path in /defailtConfig/config.json
+sed -Ei "s|/dev/tty[A-Z]*[0-9]|$device|g" $defCJ
+
+# Folders and files that we need to copy to the external /data directory
+paths="/opt/z-way-server/automation/storage /opt/z-way-server/automation/userModules /etc/zbw /etc/init.d/zbw_connect /opt/z-way-server/automation/.syscommands"
+for path in $paths; do
+  # Сheck the presence of the /data/ folder and the original server directories.
+  if [ -e $path -a ! -e /data/$path ]; then
+    # Make folders from the path variable with the exception of the last folder in the path
+    mkdir -p /data/`dirname $path`
+    # Moving files and folders from the standard directory to the /data folder
+    mv $path /data/$path
+  else
+    rm -rf $path
+  fi
+# Creating symlinks to the objects that we moved
+  ln -sf /data$path $path
+done
+
+# User configjson file path
+userCJ=$(ls /opt/z-way-server/automation/storage/configjson-*)
+
+# If we can't find configjson file in /data dirrectory we think this is first start and generate new remote ID
+if [ ! -f /data$userCJ ]; then 
+  wget 'http://find.z-wave.me/zbw_new_user?box_type=razberry' -O /tmp/zbw_connect_setup.run 
```

Comment:
This is done by zbw_autosetup

Reply:
As far as I remember, we used this to ensure that each running image had a different ID